proto/zitadel/resources/action/v3alpha/target.proto

```diff
@@ -27,21 +32,34 @@ message Target {
   }
   // Timeout defines the duration until ZITADEL cancels the execution.
   google.protobuf.Duration timeout = 5 [
+    (validate.rules).duration = {gte: {}, lte: {seconds: 20}},
```

Comment:
I would not add such a low max for the moment because debugging could get difficult. I currently dont have a better idea how we could solve the debug part.

Reply:
I raised the max to 270 seconds

Reply:
@stebenz and I discussed this.
Actually, the actions max timeout should be configurable for each ZITADEL runtime, as it influences metrics like request latencies and concurrent requests.

We decided to set it to 20 seconds max again for now (like the v1 actions), and create a follow-up issue to make the max configurable.

Reply:
thx for the fix @stebenz

Reply:
I created the follow-up https://github.com/zitadel/zitadel/issues/8426